packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -381,6 +383,14 @@ export function ArticleMetadataPanel({
                   onChange={breaking => onChange?.({...value, breaking})}
                 />
               </FormGroup>
+              <FormGroup>
+                <ControlLabel>{t('articleEditor.panels.canonicalUrl')}</ControlLabel>
+                <FormControl
+                  value={canonicalUrl}
+                  onChange={canonicalUrl => onChange?.({...value, canonicalUrl})}
+                />
+                <HelpBlock>{t('articleEditor.panels.canonicalUrLHelpBlock')}</HelpBlock>
```

Comment:
Help block should contain a link to https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls